ui/src/main/webapp/package.json

```diff
@@ -33,6 +33,7 @@
     "google-code-prettify": "1.0.1",
     "jquery": "2.1.4",
     "jquery-match-height": "0.7.0",
+    "ng2-file-upload": "^1.1.0",
```

Comment:
Do you consider letting the version float upwards to be safe for this dependency? I'm not against it, just curious if you have any ideas in mind for when to do this vs not.

For example, it was dangerous to do this during the NG2 RCs, but maybe that was just because of the NG2 team's sloppiness during that process.